frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1711,16 +1709,17 @@ function ContextMenuInner(propsWithInputRef: Props) {
   return (
     <React.Fragment>
       <Shortcut supportInputElements keys="escape" onTrigger={hideContextMenu} />
-      <Dropdown
-        menu={menu}
-        overlayClassName="dropdown-overlay-container-for-context-menu"
-        open={contextMenuPosition != null}
-        getPopupContainer={() => refContent}
-        // @ts-ignore
-        destroyPopupOnHide
-      >
-        <div />
-      </Dropdown>
+      {
+        <Dropdown
+          menu={menu}
+          overlayClassName="dropdown-overlay-container-for-context-menu"
+          open={contextMenuPosition != null}
+          getPopupContainer={() => refContent}
+          destroyPopupOnHide
+        >
+          <div />
+        </Dropdown>
+      }
```

Comment:
Aren't theser `{}` brackets unnecessary? Or does this somehow resolve the ts lint error?